### PR TITLE
Updated command set

### PR DESCRIPTION
**Pull request by Xelu86**
# PR Summary

- Updated Hyper-V Module with current information
- Cmdlet name - Changed bolded text to code spans
- Parameters - Changed italic text to bold
- Parameters - Changed code span text to bold
- Applied text contractions
- Applied syntax reflows
- Fixed indented code blocks

## PR Checklist

- [X] **Descriptive Title:** This PR's title is a synopsis of the changes it proposes.
- [X] **Summary:** This PR's summary describes the scope and intent of the change.
- [X] **Contributor's Guide:** I have read the [contributors guide][contrib].
- [X] **Style:** This PR adheres to the [style guide][style].

### CODE REVIEW

File: docset/winserver2022-ps/hyper-v/New-VHD.md [hunk 1]
```diff
@@ -16,95 +16,121 @@ Creates one or more new virtual hard disks.
...
-PS C:\> New-VHD -Path $vhdpath -Dynamic -SizeBytes $vhdsize | Mount-VHD -Passthru |Initialize-Disk -Passthru |New-Partition -AssignDriveLetter -UseMaximumSize |Format-Volume -FileSystem NTFS -Confirm:$false -Force
+
+```powershell
+$vhdpath = "C:\VHDs\Test.vhdx"
+$vhdsize = 127GB
+New-VHD -Path $vhdpath -Dynamic -SizeBytes $vhdsize | Mount-VHD -PassThru | Initialize-Disk -PassThru | New-Partition -AssignDriveLetter -UseMaximumSize | Format-Volume -FileSystem NTFS -Confirm $false -Force
 ```
 
-This example creates a new 127GB VHD and then mounts, initializes, and formats it so the drive is ready to use.
+This example creates a new 127GB VHD and then mounts, initializes, and formats it so the drive is
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/hyper-v/New-VHD.md:121]:
```suggestion
This example creates a new 127 GB VHD and then mounts, initializes, and formats it so the drive is
```

File: docset/winserver2022-ps/hyper-v/Set-VHD.md [hunk 2]
```diff
@@ -234,14 +261,14 @@ Accept wildcard characters: False
 ```
 
 ### -PhysicalSectorSizeBytes
-Specifies the physical sector size, in bytes.
-Valid values are 512 and 4096.
-This parameter is supported only on a VHDX-format disk that is not attached when the operation is initiated.
+
+Specifies the physical sector size, in bytes. Valid values are 512 and 4096. This parameter is
+supported only on a VHDX-format disk that'sn't attached when the operation is initiated.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/hyper-v/Set-VHD.md:266]:
```suggestion
supported only on a VHDX-format disk that isn't attached when the operation is initiated.
```

File: docset/winserver2022-ps/hyper-v/Test-VHD.md [hunk 3]
```diff
@@ -11,45 +11,52 @@ title: Test-VHD
 # Test-VHD
 
 ## SYNOPSIS
-Tests a virtual hard disk for any problems that would make it unusable.
+Tests a virtual hard disk for any problems that'd make it unusable.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/hyper-v/Test-VHD.md:14]:
```suggestion
Tests a virtual hard disk for any problems that would make it unusable.
```

File: docset/winserver2022-ps/hyper-v/Test-VHD.md [hunk 4]
```diff
@@ -11,45 +11,52 @@ title: Test-VHD
...
+
 ```
 Test-VHD [-Path] <String[]> [-SupportPersistentReservations] [-CimSession <CimSession[]>]
  [-ComputerName <String[]>] [-Credential <PSCredential[]>] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Test-VHD** cmdlet tests a virtual hard disk for any problems that would make it unusable.
+
+The `Test-VHD` cmdlet tests a virtual hard disk for any problems that'd make it unusable.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/hyper-v/Test-VHD.md:34]:
```suggestion
The `Test-VHD` cmdlet tests a virtual hard disk for any problems that would make it unusable.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @Xelu86, kindly commit the suggested changes. Many thanks.

**Review by get-itips — APPROVED** [review]

**Comment by get-itips** [comment]
Are any of these changes also applicable to other OS versions of the articles? if so, can you also update them? Thank you

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@Xelu86 hope all is well. Could you please submit the changes?

**Comment by scanum** [comment]
We appreciate any feedback that improves the content in the Microsoft docs but we haven't had any response from you so we are going to close this issue.  
 
Please feel free to re-open this issue if there is a specific area of the docs that we can improve or make better. Thank you.

**Pull request closed without merging**